apps/api/src/app/app.module.ts

```diff
@@ -93,6 +95,17 @@ const autoSchemaFile = environment.production
         xroadPathV2: environment.drivingLicense.v2.xroadPath!,
       },
     }),
+    // DrivingLicenseBook has drivingIstructorGuard that uses drivingLicenseService
+    // DrivingLicenseBookModule needs to register DrivingLicenseModule and uses the same config to do so
+    DrivingLicenseBookModule.register({
+      clientConfig: {
+        xroadBaseUrl: environment.xroad.baseUrl!,
+        xroadClientId: environment.xroad.clientId!,
+        secret: environment.drivingLicense.secret!,
+        xroadPathV1: environment.drivingLicense.v1.xroadPath!,
+        xroadPathV2: environment.drivingLicense.v2.xroadPath!,
+      },
+    }),
```

Comment:
We are under some pressure to get this through and get going with `DrivingTeacherBooking`. Do you mind me adding the refactoring to our internal sprint backlog and deal with it for the next release?

Reply:
A simplification of the DrivingLicenseModule is a good idea, we will implement it in a follow up PR, since it effects the Driving License application more and would be nice to do it in a clean PR.

Reply:
Agreed.